### ISSUE TITLE
assigning component lists to teams

### ISSUE THREAD

**Issue by tomkolp** [WeblateOrg/weblate#11549]
### Describe the problem

Currently, it is not possible to assign component lists to teams.  The django admin interface allows you to assign component lists to groups.  To resign from the django interface, you must introduce this functionality.

### Describe the solution you would like

Added functionality for assigning component lists to teams.

**Comment by nijel** [comment]
Yes, the whole component lists need to be revisited because now they can be only changed in Django admin interface and that is also the only place they can be used in teams.

**Comment by nijel** [comment]
Eh, actually it is possible in site-wide teams management (as can be seen in https://github.com/WeblateOrg/weblate/issues/11536). That is currently in line with fact that component lists are site-wide feature only.
